yt/data_objects/time_series.py

```diff
@@ -418,6 +418,8 @@ def particle_trajectories(self, indices, fields=None, suppress_logging=False):
         suppress_logging : boolean
             Suppress yt's logging when iterating over the simulation time
             series. Default: False
+        ptype : str, optional
+            Only use this particle type
```

Comment:
We should say something like `Default: None, which uses all particle types`. Also, can you put this element of the docstring in the `ParticleTrajectories` class itself?